src/transform/schema-object.ts

```diff
@@ -144,7 +144,7 @@ export function defaultSchemaObjectTransform(
           return ctx.immutableTypes || schemaObject.readOnly ? tsReadonly(itemType) : itemType;
         } else {
           return tsUnionOf(
-            ...Array.from({ length: (maxItems || 0) - minItems + 1 })
+            ...Array.from({ length: (maxItems ?? 0) - minItems + 1 })
```

Comment:
I’m so happy to finally be able to use this. For the longest time I avoided it purely out of Node 12’s lack of support :/